### DISCUSSION TITLE
xsearch path not available on LC

### DISCUSSION THREAD

**Discussion by ShihengDuan** [xCDAT/xcdat#424] [2. Questions] [answered]
I'm following the notebook for xsearch: 
https://github.com/xCDAT/xcdat/blob/main/docs/demos/1-25-23-cwss-seminar/xsearch-xcdat-example.ipynb

It mentioned that on LC machines, the path is ```/p/climate/user_pub``` but it seems not exist. 

After installing from GitHub, I cannot search for any CMIP data. Does that mean the CMIP data is only available on Gates? 

Thanks!

**Comment by pochedls** [comment, marked as answer]
@ShihengDuan – [xsearch](https://github.com/pochedls/xsearch/) is not supported by xcdat – but I am seeing it on quartz. Feel free to email me if you have questions about this.

**Reply by ShihengDuan** [reply]
Thanks I can access it now. There might be a typo in the demo notebook: should it be ```/p/climate/user_pub/xclim/persist/software/xsearch``` instead of ```/p/climate/user_pub/xclim/persist/software/xcdat/``` in Section 2 Step2?

**Reply by pochedls** [reply]
That looks right! Thanks. I'll see if I can make that modification...

**Comment by jypeter** [comment]
Note that IPSL and CNRM have developed  similar tools to work with CMIPn mirrors and IPSL/CNRM output data, if this can be a source of ideas/features/... : [CliMAF - a Climate Model Assessment Framework](https://climaf.readthedocs.io/)
